libethereum/State.cpp

```diff
@@ -530,23 +539,22 @@ void State::revert(size_t _savepoint)
 		// change log entry.
 		switch (change.kind)
 		{
-		case Change::storage:
+		case Change::Storage:
 			account.setStorage(change.key, change.value);
 			break;
-		case Change::balance:
+		case Change::Balance:
 			account.addBalance(0 - change.value);
 			break;
-		case Change::nonce:
+		case Change::Nonce:
 			account.setNonce(m_cache[change.address].nonce() - 1);
 			break;
-		case Change::create:
-			account.kill();
-			break;
-		case Change::prefund_create:
-			// FIXME: add prefound and CREATE revert in single transaction.
+		case Change::Create:
+			// Clear the deployed code (if any) and mark as not dirty.
+			// FIXME: probably wrong for accounts prefunded in the same
+			//        transaction. Separate Create and SetCode events.
 			account.setCode({});
```

Comment:
Previously it was account.kill() for Change::Create
So where do we kill the account now?